manual/src/html_processing/scss/manual.scss

```diff
@@ -283,7 +283,6 @@ blockquote.quote{
 span.c003{
     color:#564233;
     font-family:"Fira Mono",courier;
-    background-color:#f3ece6;
```

Comment:
What is the motivation behind the change?

Reply:
The PDF and old HTML manuals don't color the background of inline code. 

In cases where you have syntax fragments interleaved with eg. grammar rules, this kind of styling can make the whole thing look incoherent, especially when they are inside a paragraph. You can take look at "[Local modules](https://ocaml.org/manual/expr.html)". Maybe it would be nice if we could make the whole construct "let module module-name =  module-expr in  expr" have a colored background with mixed typewriter and italic font inside, but we can't do it now since it's not a single HTML element.

Reply:
The incoherence is a good point, thanks for pointing that out.